VideoWeb/VideoWeb/ClientApp/src/app/shared/cookie-banner/cookie-banner.component.html

```diff
@@ -0,0 +1,24 @@
+<div *ngIf="isBannerVisible" class="cookie-container govuk-cookie-banner" data-nosnippet role="region" aria-label="Cookies on Video Hearing Service">
```

Comment:
the parent component `app.component` should determine whether or not to display the component. If the component is added, but the content in the browser is empty it would require more digging than necessary to understand why. The parent component should be responsible for hiding/showing child components like this

Reply:
done